### PR TITLE
Option gui hidepanels

### PR DESCRIPTION
**Pull request by tanguyduval**
Could not see all the buttons in qmt_spgr (compute table and help). Also the panels hidding in Custom_OptionGUI is simplified.

### CODE REVIEW

File: src/Models/FieldMaps/b1_dam.m [hunk 1]
```diff
@@ -73,4 +73,17 @@
         end
         
     end
-end
+    
+    
+    methods(Access = protected)
```
Comment (mathieuboudreau) [inline src/Models/FieldMaps/b1_dam.m]: @agahkarakuzu could you review this PR? I think the changes in this file might conflict with the changes we're about to merge in your older B1dam PR (#290)

File: src/Common/GUI/Custom_OptionsGUI.m [hunk 2]
```diff
@@ -176,52 +176,34 @@ function OptionsGUI_OpeningFcn(hObject, eventdata, handles, varargin)
         warning('Problem with adding TooltipString');
 
     end
+else
+    set(handles.FitOptTable,'ColumnName',{'Variables'})
+    set(handles.FitOptTable,'ColumnFormat',{'char'})
+    set(handles.FitOptTable,'ColumnWidth',{100})
+    set(handles.FitOptTable,'Data',Model.xnames')
```
[the remark this answers is not in the document]
Comment (tanguyduval) [reply src/Common/GUI/Custom_OptionsGUI.m]: See my answer in #272 I reverted this change

File: src/Models/FieldMaps/b1_dam.m [hunk 3]
```diff
@@ -45,7 +45,7 @@
 
     properties
         MRIinputs = {'SF60','SF120'};
-        xnames = {};
+        xnames = {'B1map'};
```
[the remark this answers is not in the document]
Comment (tanguyduval) [reply src/Models/FieldMaps/b1_dam.m]: done

**Comment by agahkarakuzu** [comment]
These changes are outdated. Closing this PR.

**Pull request closed without merging**